src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -59,7 +59,7 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
                 new PostgresTablesTreeItem(this, clientConfig)
             ];
 
-            if (this.parent.supportsStoredProcedures()) {
+            if (this.parent.supportsStoredProcedures(clientConfig)) {
```

Comment:
You have to await this since it's async now. Otherwise it returns the promise directly, which is always a truth-y value. I thought we had tslint for this, but might not be enabled in this repo